quinn-proto/src/iroh_hp.rs

```diff
@@ -40,13 +40,13 @@ pub enum Error {
 }
 
 pub(crate) struct NatTraversalRound {
-    /// Sequence number to use for the new reach out frames
+    /// Sequence number to use for the new reach out frames.
     pub(crate) new_round: VarInt,
-    /// Addresses to use to send reach out frames
-    pub(crate) reach_out_at: Vec<(IpAddr, u16)>,
-    /// Remotes to probe by attempting to open new paths
-    pub(crate) addresses_to_probe: Vec<(IpAddr, u16)>,
-    /// [`PathId`]s of the cancelled round
+    /// Addresses to use to send reach out frames.
+    pub(crate) reach_out_at: Vec<IpPort>,
+    /// Remotes to probe by attempting to open new paths.
+    pub(crate) addresses_to_probe: Vec<(VarInt, IpPort)>,
```

Comment:
can you add a comment what the `VarInt` is

Reply:
done in 0b93d82c